src/main/webapp/app/exercises/quiz/manage/drag-and-drop-question/drag-and-drop-question-edit.component.scss

```diff
@@ -130,4 +130,48 @@
             width: 100%;
         }
     }
+
+    .question-card-header {
+        width: 100%;
+        padding: 15px 5px 5px 5px;
+
+        .row {
+            width: 100%;
+        }
+
+        .question-collapse {
+            margin-top: -15px;
```

Comment:
Set other margins differently instead of using negative margins, as that can lead to unwanted side effects

Reply:
Removed this and adjusted other margins.